specification/network/resource-manager/Microsoft.Network/preview/2024-03-01/webapplicationfirewall.json

```diff
@@ -783,6 +783,14 @@
           "items": {
             "$ref": "#/definitions/ManagedRuleGroupOverride"
           }
+        },
+        "computedDisabledRuleGroups": {
```

Comment:
The field computedDisabledRuleGroups may not make much sense to a user who is unaware of default disabled rules.

Reply:
Would it be more user-friendly if we rename it to `DisabledRuleGroups`? One potential downside is that the discrepancy between Disabled Overrides and `DisabledRuleGroups` might be confusing. 
As an alternative, we could consider the name `EffectiveDisabledRuleGroups`